lib/tasks/default.rake

```diff
@@ -427,6 +427,62 @@ if Rails.env.development?
   end
 end
 
+require 'open-uri'
+# Returns "true" if okay
+# rubocop:disable Metrics/MethodLength
+def validate_one_link(link)
+  puts "  <#{link}>"
+  return false if link.blank?
+  # '%{..}' is used when we generate URLs, presume they're okay.
+  return true if link.start_with?('mailto:', '/', '#', '%{')
+  # Shortcut: If we have anything other than http/https, it's wrong.
+  return false unless link.start_with?('https://', 'http://')
+  return false if link.include?(' ') # Common problem
+  begin
+    code = open(link).status[0]
+    return code == '200'
+  rescue
+    return false
+  end
```

Comment:
It looks like the above 6 lines would be cleaner as:
```ruby
Net::HTTP.get_response(URI.parse(link)).code == '200'
```
according to https://stackoverflow.com/a/18635307/1935918

and replace `require 'open-uri'` with `require 'net/http'`.